spec/features/enter_name_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './app'
+require "capybara/rspec"
+feature 'Enter Name' do
+  scenario 'player submits name' do
+    visit ('/')
+    #give them a form to fill in
+    fill_in 'name', with: 'Aysha'
+    # give them  button to submit
+    click_button 'submit and play'
+    #display thier name on the screen
+    #save_and_open_page
+    expect(page).to have_content('Aysha')
+  end
+end
+
+feature 'Choose your weapon' do
+  scenario 'player submits weapon' do
+  visit ('/outcome_final')
+  fill_in 'weapon', with: 'Rock'
+  click_button 'submit and play'
+  expect(page).to have_content('Rock')
+end
+end
+#feature 'Display outcome' do
+  #scenario 'display who wins' do
+  #visit ('/outcome_final')
+  #fill_in 'weapon', with: 'Rock'
```

Comment:
Looks like you're very close with this test. What would be the steps for your manual test?